### PR TITLE
[WIP] Update quintanalibre.org.ar to LibreRouterOs v1.1

### PR DESCRIPTION
**Pull request by spiccinini**

### CODE REVIEW

File: quintanalibre.org.ar/comun/etc/dropbear/authorized_keys [hunk 1]
```diff
@@ -6,4 +6,8 @@ ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDIBva1qwYBMwQThY57+2OKUpld8BC7s3RruAj7hnhM
...
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDDmmm+KDqSsZkvgbY3Mm2Zdr+0KmVWjBPdMMktc1tmvna28xP/kUXfzy4Z0JStkCBl/9wWXe7QYPZpoyoL2i9uSrFlZoWRxvcGS/HGCnS7F33YCyoSNnojR8nSiW2KDmLLODIEUsh/btbgQXbo92hDMAgJIPl/GckQeRn2RqxUoOdOhXD1OJ0q9Zpc3wSHQfPd9i7hYztYs5+J/pFiBU/q/71B2VWCdjC8EwzdJmEEErx6h3eX7MMFITNtQtXjsgCkFUDlLQ6xI3WMmEF9ks6Bgw60C0LCsjXRC1EZtba2r26agrPKKWfjqMjcLHwgUIaot5oFjexgvv17JuyJsV6J san@bonzo
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDh9wTTSbhjNMnfZM2iiKvPmkAzXznYQUc6al59uNCtVOSEI/Z5LC+pNGmrkG8LqfKrnMpnH7/xgTKNkz+GeTlJCTWlG5pGBcB371qNMekdk2V2Kr+z2mRofuoelsEs7+aErklJwb6kGsj46vwXmEtpTQNLiFQcHvU+xCsTIlxbjkqo2zAhDk0mlz929jASaMzvQOTYKN6VCBQMy5gBUbwGIV/lXE04TOztDdFae2XjLaYp+1zEwx6IoPv8b7XEQlMP2PsTi33KO00E3wdgptJrLmTPe9RrKMY+yxxBU3Vyno4S04vr90MpK1wqq42WSwGB6J17N+SxkBx47B5VfLL7 dbellomo@newen
```
Comment (germanferrero) [inline quintanalibre.org.ar/comun/etc/dropbear/authorized_keys]: Are we sure about this key? AFAIK he has not been in touch with the community for la long time

File: quintanalibre.org.ar/comun/etc/uci-defaults/92-cron-workaround-tasks [hunk 2]
```diff
@@ -6,28 +6,6 @@ internet_gw_ip=10.5.24.12
...
-	sed "\| \* \* \* .*regular scan.*|d" -i /etc/crontabs/root
-echo "*/5 * * * * killall iw; logger -t workarounds 'regular scan on wlan1'; iw wlan1-adhoc scan freq 5745 5825 &" >> /etc/crontabs/root
-echo "*/5 * * * * killall iw; logger -t workarounds 'regular scan on wlan0'; iw wlan0-adhoc scan freq 2412 2457 &" >> /etc/crontabs/root
-
-# si dnsmasq está muerto, lo reiniciamos. Chequeamos que haya acceso a Internet antes de intentar resolver.
-sed "\| \* \* \* .*dnsmasq is alive.*|d" -i /etc/crontabs/root
-echo "*/8 * * * * if ping -c5 -w10 8.8.8.8; then if nslookup google.com 127.0.0.1 &>/dev/null; then logger -t workarounds 'OK. dnsmasq is alive'; else logger -t workarounds '*dnsmasq* FAIL. name resolution unsuccessful. Restarting the process'; /etc/init.d/dnsmasq stop; killall -9 dnsmasq; /etc/init.d/dnsmasq start; logger -t workarounds 'restarted zombie dnsmasq'; fi; else logger -t workarounds 'skipping dnsmasq check because Internet is unreachable'; fi" >> /etc/crontabs/root
```
Comment (germanferrero) [inline quintanalibre.org.ar/comun/etc/uci-defaults/92-cron-workaround-tasks:27]: ql-nicojesigioia has this crontask enabled at the moment. Just garbage?

Comment (spiccinini) [reply quintanalibre.org.ar/comun/etc/uci-defaults/92-cron-workaround-tasks:27]: I beleave that dnsmasq crashes are fixed now as I don't see "restarted zombie dnsmasq" in the logs....Let's test it without the workarround (yes we need the feature that reports this kind of metrics)

File: quintanalibre.org.ar/comun/etc/config/lime-community [hunk 3]
```diff
@@ -0,0 +1,30 @@
+config lime system
```
Comment (germanferrero) [inline quintanalibre.org.ar/comun/etc/config/lime-community]: Should we add this same file at librerouter directory?

**Review by germanferrero — APPROVED** [review]

File: quintanalibre.org.ar/generic/files/etc/config/lime-community [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	option channel_2ghz '11'
+
+	list channel_5ghz '136'
+	list channel_5ghz '60'
+
+	option distance_2ghz '100'
+	option distance_5ghz '1000'
+
+	list modes 'ap_2ghz'
+	list modes 'apname_2ghz'
```
Comment (G10h4ck) [inline quintanalibre.org.ar/generic/files/etc/config/lime-community:32]: el `_2ghz` no va apegado a modes? o justo esta opcion la gestionamos distinto?

File: quintanalibre.org.ar/generic/files/etc/dropbear/authorized_keys [hunk 5]
```diff
@@ -6,4 +6,7 @@ ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDIBva1qwYBMwQThY57+2OKUpld8BC7s3RruAj7hnhM
...
...aHxR9pt1uavuifCu4cYb8Xm9rg0pDNpS+CQ1P/MYRGVPOE47wGMvqseCDZL8ehJLNNEYnh39lD4foent3ELHzxGi2UJPXWcfWipg/O8GXE1qqU0UXzyzTigZgoJRR/9AyqmXfXI8qmv7GCfJA6rt9eS4PWBN/dc9TEesn1xxH7DvqUQmHQq+xjFbCukegwJlu8dt5HGsKXiNxVBvf43DESLvaQNarBkQDQfN4ZhAUYlDdK6hpHDpIHcvVS4IzU/Y1wGXpOAXVvXNwf1w37Wuitiy3A6pF8guagQ46v4oQ/r8nXrB00WoaCCJNMP2ywoVpwRX+AoxfdTIcZ/2UNcx5eF/8ib2aaaE5JH5Svo08/1qIyEZTQ1ReaZFGut1hUwtEUTjejUft/6pOpcsaiXfo3WXs1cdfozoLoEwx5+x1oyRpw+O+skX+K/BvQmQhLm3AGMck/CnAFH06E15KKa0EsI5vc2YsHnbnL7DDJyR9KWpoOTAklx0e3c0zyNAwW6J+y3v1d74fPteB0db/acyIULzogXIoL5/4KYh7L6VqQ/C+OxPTSgjyq2Ze+ipkObHoHMrP6NZ1Bi9xkvKkcWZN3YbxonsbYjBVCGUWV4obo7eW5FPEA/rsToUa4uZPccNjop+lGKLfK4BKSzgQ9fpFVnjnkzI9nTM18DoG3DB5c3nMXbX8ORB/AZwB2EMBO5/qO93YPiksjraPIfcXkHBxsfXJGk8yId9xxblQWciiyO+rzmUsMvt5vDfEu6YXYDCdvHSnLX2mzuYj5CseLH2L2/aw+6ny0CSHqbCLULTacynat0xO6PRxf0Pk/Yia2KtJzZQ55sG9QKEYS2OWXsSVhVG2to4Zf7/xqc8BQXLyW1cocMPiFQUNxyXcXfiy5OwODjk6etXrE0XiSU9QLlgPFGzqctEO3uF+xRAiCNp3stnZBcOPQz8vzw3D6/rdeN3fbqj5GHCeKmAq8DQ3rvUxohEmEfP0Y58gLZfW1u4qS2oEconD7hN0CEEPEPwMULaao5+KXs4gclG2vTAF7v8baTCxO2Pbf+7BCq22BDG5E2ILxuxEca4e2NxKwXHrb6jUek8jDuYem5qHArYOY1SVCrMyqgkSsW0biLtDBNFqAhKnIhN7wIs6bXkRWsRP/NBco06jGd9blt6QExz1OorQxxc++5axzGWHElQl p4u@nomada
```
Comment (G10h4ck) [inline quintanalibre.org.ar/generic/files/etc/dropbear/authorized_keys]: esta creo que se puede quitar

**Review by G10h4ck — COMMENTED** [review]
See comments inline

File: quintanalibre.org.ar/generic/files/etc/config/lime-community (hunk 4 above)
Comment (spiccinini) [reply quintanalibre.org.ar/generic/files/etc/config/lime-community:32]: nope, el `_2ghz` va al final asi como está anda bien (y esta asi en el lime-example)

**Pull request merged**